app/views/topical_events/show.html.erb

```diff
@@ -16,8 +16,14 @@
       margin_bottom: 8,
     } %>
 
-    <% if @topical_event.image_url %>
-      <%= image_tag(@topical_event.image_url, alt: @topical_event.image_alt_text) %>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <% if @topical_event.organisations %>
+      <%= render "govuk_publishing_components/components/metadata", {
+        other: {
+          "Organisations": sanitize(array_of_links_to_organisations(@topical_event.organisations).to_sentence)
+        }
+      } %>
```

Comment:
I notice there is [a partial](https://github.com/alphagov/collections/blob/ae77f463a80aaec1ee27f9b43e9d4d6f64422670/app/views/roles/_organisations.html.erb) (used solely by roles at the moment) which does exactly this.

Maybe we could make that partial shared so the same code isn't duplicated across multiple document types?